package.json

```diff
@@ -9,31 +9,31 @@
     "test": "jest"
   },
   "dependencies": {
-    "@apollo/client": "3.4.16",
-    "@testing-library/jest-dom": "5.14.1",
-    "@testing-library/react": "12.1.2",
-    "@types/cookie": "0.4.1",
-    "@types/jest": "27.0.2",
-    "@types/lodash.isequal": "4.5.5",
-    "@types/react": "17.0.32",
+    "@apollo/client": "3.5.5",
+    "@testing-library/jest-dom": "5.16.0",
```

Comment:
Hi! Thank you for your pull request!

Why won't we move `@testing-library/jest-dom` to the devDependencies as `@testing-library/react`?

Reply:
Because I forgot it ;)